resources/client scripts/index.js

```diff
@@ -0,0 +1,36 @@
+fetch('http://localhost:3000/users')
```

Comment:
Please, define an `async` function to do this task, and call directly in the page's HTML.

Also, it's better to avoid using spaces in paths.